Modelica/Electrical/Analog/UsersGuide/package.mo

```diff
@@ -0,0 +1,23 @@
+within Modelica.Electrical.Analog;
+package UsersGuide "User's Guide"
+  extends Modelica.Icons.Information;
+  annotation (
+    preferredView="info",
+    DocumentationClass=true,
+    Documentation(info="<html>
+<p>
+The Analog package contains the following components:
+</p>
+
+<ul>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Basic\">Basic</a>: basic components (ground, resistor, capacitor, conductor, inductor, transformer, gyrator)</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Ideal\">Ideal</a>: ideal elements (switches, diode, transformer, idle, short, ...)</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Lines\">Lines</a>: transmission lines (lossy and lossless)</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Semiconductors\">Semiconductors</a>: semiconductor devices (diode, bipolar and MOS transistors)</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Interfaces\">Interfaces</a>: ideal elements (switches, diode, transformer, idle, short, ...)</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Sensors\">Sensors</a>: sensors to measure potential, voltage, current and power</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Sources\">Sources</a>: time-dependent and controlled voltage and current sources</li>
+<li><a href=\"modelica://Modelica.Electrical.Analog.Icons\">Icons</a>: domain specific icons</li>
+</ul>
```

Comment:
The new Batteries sub-library is missing in that list.

Reply:
The Batteries library will be included in Electrical.Batteries, so I did not consider it here.

Reply:
Currently it is in Electrical.Analog,